### ISSUE TITLE
PUDGE  Phase Boots 5 years bug FIX PLEASE

### ISSUE THREAD

**Issue by vermillionXD** [jeffhill/Dota2#518]
### Item name

Phase Boots

### Description

When ur Phase Boots on pudge is on CD, in russian localisation, he everytime saing "not yet" and its imposible to hear every time. This bug in dota like 5 -6 years. FIX PLEASE

### Example Match ID and Timestamp

Lobby pudge,russian voise,Phase Boots

**Comment by art3mis1smz** [comment]
i dont think that be gonna bug, beacuse on lots of heroes you will hear im not ready or not time yet thats just for tell you yeah bro im on cooldown lets try next few seconds

**Comment by ZharDar** [comment]
I think he means that it constantly plays the sound effect, even while just walking.

> i dont think that be gonna bug, beacuse on lots of heroes you will hear im not ready or not time yet thats just for tell you yeah bro im on cooldown lets try next few seconds

**Comment by vermillionXD** [comment]
Yes. Fix please
 
 
--
Антон Танский
Отправлено из Почты  Mail.ru
